packages/core/src/traced-orchestrator.ts

```diff
@@ -3,8 +3,7 @@ import { ArbitraryObject, Maybe } from '@envelop/types';
 import { EnvelopOrchestrator } from './orchestrator';
 import { isAsyncIterable } from './utils';
 
-const getTimestamp =
-  typeof globalThis !== 'undefined' && globalThis?.performance?.now ? () => performance.now() : () => Date.now();
+const getTimestamp = typeof performance !== 'undefined' && performance.now ? () => performance.now() : () => Date.now();
```

Comment:
```suggestion
const getTimestamp = globalThis?.performance?.now ? () => performance.now() : () => Date.now();
```

I dont think we need a typeof check here globalThis should be everywhere

Reply:
Makes sense!